quiz.js

```diff
@@ -49,8 +72,7 @@ db.dropDatabase();
 
 // your code begins here
 
-
-
+db.motto.remove( { motto : "As high as honor" }, House Arryn )
 
```

Comment:
Nope. First, we don't have a collection called 'motto', so calling `db.motto` doesn't get us anything. Second, we don't want to delete House Arryn, just remove one of its properties; the right tool for this job is actually `update`, with an $unset operation.

Here's the syntax:
```
db.houses.update({house : 'Arryn' }, { $unset : { motto : "" } });
```
Although we're not actually assigning motto a new value, Mongo expects key-value pairs to always be matched, so we need to specify a value along with `motto`; this value, however, will be ignored.